dev-docs/bidders/ix-server.md

```diff
@@ -66,11 +68,72 @@ The following table lists the media types that Index supports. For information a
 | video   | Supported, including ad pods for OTT    |
 | native      | Supported       |
 
-<a id="call-index"></a>
+<a id="hosting-instance"></a>
+
+## Configure the Index adapter in your Prebid Server instance 
+
+**Before you begin:** Contact your Index Exchange Representative to get an endpoint and setup instructions.
+
+If you are hosting your own Prebid Server instance, depending on whether you are using Prebid Server Go or Prebid Server Java version, complete one of the following steps: 
+
+* If you are using the Prebid Server Go version, in the `static/bidder-info/ix.yaml` file, complete the following:
+
+  * Enable the adapter by deleting the `disabled: true` entry.
+  * Add the following new entry and include the regional endpoint provided to you by Index:
+
+    ```javascript
+     endpoint: "https://<ENDPOINT URL>"
+     ```
+
+  * Edit the below existing entry and include your publisher ID in the `s` parameter:
```

Comment:
@rimaburder-index - in general, this is a nice improvement - thanks. I re-titled the section headings and hope you find that update an improvement.

I would like to question the use of the word "publisher" here. Many companies that host Prebid Server are Ad Tech companies and not really publishers, so I would suggest changing "publisher ID" to "partner ID" for the purposes of this documentation?

Reply:
@bretg really like the title suggestion. Your Partner ID update is also a good suggestion and we're going to review it as part of our next docs update.